src/OceanSeaIceModels/InterfaceComputations/interface_states.jl

```diff
@@ -103,7 +103,7 @@ function WaterMoleFraction(FT=Oceananigans.defaults.FloatType)
         magnesium = SalinityConstituent{FT}(24.31, 0.05),
     )
 
-    return SeawaterComposition(water_molar_mass, salinity_constituents)
+    return WaterMoleFraction(water_molar_mass, salinity_constituents)
```

Comment:
what's this change?

Reply:
it's a bugfix (`SeawaterComposition` does not exist)